lib/src/shared_widgets/menu_button.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:mustang_viewer/src/utils/app_styles.dart';
+
+class MenuButton extends StatelessWidget {
+  const MenuButton({
+    Key? key,
+    required this.buttonIcon,
+    required this.tooltipMessage,
+    required this.onPress,
+    this.selected = false,
+  }) : super(key: key);
+
+  final Icon buttonIcon;
+
+  final String tooltipMessage;
+
+  final bool selected;
```

Comment:
will `active` be more accurate instead of `selected` ?

Reply:
true